### ISSUE TITLE
Support for guest-comments

### ISSUE THREAD

**Issue by rajivharlalka** [utterance/utterances#666]
I have my blog setup with using utterances as my comment manager. Is there an option of having guest comments where person doesnt need to login using Github to add a comment?

**Comment by a1603169** [comment]
I don't think that is possible unless Utterance supports different auth rather than Github.
